### PR TITLE
R1.9.1

### PR DESCRIPTION
**Pull request by miaoshasha**
create tests for 1.9.1 version, mock, for checking if tests can be correctly created. will delete later
the 1.9.1 wheels are incorrectly named as 1.9, but notice the build date 09/23. so they actually are 1.9.1 wheels (copies at build time).

### CODE REVIEW

**Comment by miaoshasha** [comment]
testing locally with:
scripts/run-oneshot.sh --accelerator v2-8 --file tests/tensorflow/nightly/mnist.libsonnet --type functional

File: tests/pytorch/experimental.libsonnet [hunk 1]
```diff
@@ -184,4 +184,73 @@ local experimental = import '../experimental.libsonnet';
...
     },
   },
+  PyTorch1_9_1TpuVmPodTest:: experimental.BaseTpuVmMixin {
+    local config = self,
+    tpuSettings+: {
+      softwareVersion: 'v2-nightly',
+      tpuVmStartupScript: |||
+        sudo bash /var/scripts/docker-login.sh
+        sudo docker rm libtpu || true
+        sudo docker create --name libtpu gcr.io/cloud-tpu-v2-images/libtpu:pytorch-1.9.1 "/bin/bash"
```
Comment (jysohn23) [inline tests/pytorch/experimental.libsonnet:194]: This image doesn't seem to exist?

**Comment by miaoshasha** [comment]
local run of oneshot script succeeds, job suffix 5k7bn. @jysohn23

**Comment by jysohn23** [comment]
Could you check with Will where the `tpuVmStartupScript` logs are? We should verify that `libtpu` from pypi is being installed properly. This is probably how you want to install with extra dependencies on libtpu by asking to optionally install with `tpuvm` support.

**Pull request closed without merging**